docs/process.md

```diff
@@ -53,7 +53,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
 
 4. Add remote names for committer repo (let's call it `upstream`)
    ```
-   git remote add   upstream      https://github.com/TEAMMATES/repo.git
+   git remote add   upstream      https://github.com/TEAMMATES/teammates.git
```

Comment:
Can you revert this one? Apologies, it slipped my mind before when commenting on the issue thread.